docs/source/building/meta-yaml.rst

```diff
@@ -597,11 +641,16 @@ need to put the same selector on multiple lines.
      url: http://path/to/unix/source        # [unix]
      md5: 88510902197cba0d1ab4791e0f41a66e  # [unix]
 
+
+NOTE: The fully rendered recipe file will be written back to the directory as ``meta.yaml.rendered``.
```

Comment:
more specifically, it's in the info/recipe folder.  This folder does not exist if build/include_recipe is False.